pkg/verify/verifyutil/util.go

```diff
@@ -0,0 +1,18 @@
+package verifyutil
```

Comment:
Can this just be `util` under `pkg/verify/util`?  Folks who need to distinguish from another `util` can always:

```go
import (
  verifyutil ".../pkg/verify/util"
)
```

Reply:
Fixed in [2bde02f](https://github.com/openshift/cluster-version-operator/pull/413/commits/2bde02f1bc058d9b91c9b5cce3afb3259523b74c)